iina/Base.lproj/Localizable.strings

```diff
@@ -109,6 +109,9 @@
 
 "track.none" = "<None>";
 
+"subtrack.no_loaded" = "No subtitle loaded";
```

Comment:
"Subtitles" is generally plural when used in this sense.

Reply:
Thanks! Just curious, since I was taught usually No + noun singular form, why we should use plural here?

Reply:
It's just convention; I'd assume it's because you never really just have one subtitle. Kind of like train tracks? See dict://subtitle. By the way, don't you need to update every instance of this, rather than just the ones you've done here?

Reply:
So it's because "subtitle" itself is usually in plural form, right?
>  By the way, don't you need to update every instance of this, rather than just the ones you've done here?

Those ones were generated by script, so I forgot them, sorry!